elytron/src/main/java/org/wildfly/extension/elytron/Capabilities.java

```diff
@@ -102,7 +102,7 @@
     static final String KEY_MANAGERS_CAPABILITY = CAPABILITY_BASE + "key-managers";
 
     static final RuntimeCapability<Void> KEY_MANAGERS_RUNTIME_CAPABILITY =  RuntimeCapability
-            .Builder.of(KEY_MANAGERS_CAPABILITY, true, KeyManager[].class)
+            .Builder.of(KEY_MANAGERS_CAPABILITY, true, InjectedValue.class) // InjectedValue<KeyManager[]>
```

Comment:
TBH I am not completely sure about this one - by switching to InjectedValue we are loosing the capability typing.  I think we may need to see if we have alternatives that can avoid loosing the type information.